include/lmms_basics.h

```diff
@@ -97,13 +97,13 @@ inline float typeInfo<float>::minEps()
 }
 
 template<>
-inline bool typeInfo<float>::isEqual( float _x, float _y )
+inline bool typeInfo<float>::isEqual( float x, float y )
 {
-	if( likely( _x == _y ) )
+	if( unlikely( x == y ) )
```

Comment:
It seemed to me that them being equal was more the unlikely case, but it looks to me like a premature optimization anyhow. I'll rather take it out if nobody objects.